completions/README.md

```diff
@@ -1,12 +1,18 @@
-# Shell completions
+# quick-lint-js completions
 
-To get automatic completions for quick-lint-js's options you can install the provided shell completions.
+## Using cmake
 
-## Zsh
+```sh
+cmake .
+make
+sudo make install
```

Comment:
Blocking: We shouldn't provide instructions on *only* building and installing completion files. We should install completion files as part of the CLI build. This is how the manual (`quick-lint-js.1`) is installed, for example.

(The manual installation instructions below are good to have.)